### ISSUE TITLE
site-install db parameters don't match respective drush parameters

### ISSUE THREAD

**Issue by klonos** [backdrop-contrib/b#160]
I had this drush command that works fine, and the site installation completes _unattended_ (which is the goal, as I'm using it in a shell script):

`drush si standard --db-url=mysql://root:backdrop@localhost/backdrop --site-name=some-name`

I was expecting bee to be a drop-in replacement, so I've tried it but it doesn't work as expected (it prompts me for db name and username/password):

```
bee si standard --db-url=mysql://root:backdrop@localhost/backdrop --site-name=some-name
Database name: backdrop
Database username: root
Database password: backdrop
Database host [localhost]: 
Database prefix: 
Admin username [admin]: 
Admin password [c1575ea781555c613915]: 
Admin email [admin@example.com]: 
Site mail [admin@example.com]: 
Language code [en]: 
Install profile [standard]: 
Backdrop installed successfully.
```

If I try `--db=` instead of `--db-url=`, it seems to pick it up, but then asks for the db prefix

```
bee si standard --db=mysql://root:backdrop@localhost/backdrop --site-name=some-name
Database prefix:
...
```

Then I tried setting the db prefix to blank, but it still prompts me for it:

```
bee si standard --db=mysql://root:backdrop@localhost/backdrop --site-name=some-name --db-prefix=""
Database prefix:
...
```

...or:

```
bee si standard --db=mysql://root:backdrop@localhost/backdrop --site-name=some-name --db-prefix=''
Database prefix:
...
```

**Comment by ghost** [comment]
Have you run `bee help install` to see the different options available? Specifically `--auto`...

Also note (as per the README) that Bee is "inspired by Drush", not a direct replacement.

**Comment by klonos** [comment]
> Have you run `bee help install`...

Ironically that command throws this error 😅 :
```
Trying to access array offset on value of type bool
 /usr/local/bin/bee/includes/render.inc:355
```

> Specifically `--auto` ... Bee is "inspired by Drush", not a direct replacement.

Aha! It seems backwards UX-wise to me (usually optional parameters in command-line tools are skipped by default), but if that was a conscious decision, then it's fine. I was expecting it to be a drop-in replacement for the sake of easy adoption, but I can adapt. Thanks 👍🏼

**Comment by klonos** [comment]
Things work as expected now that I know better. This issue can be closed - I've raised #161 for the error I get when running `bee help install`.
